packages/suite-desktop/src-electron/libs/http-receiver.ts

```diff
@@ -138,6 +141,7 @@ export class HttpReceiver extends EventEmitter {
      * Entry point for handling requests
      */
     private onRequest = (request: http.IncomingMessage, response: http.ServerResponse) => {
+        console.log('request.url', request.url);
```

Comment:
forgotten?

Reply:
Removed in 07c05e195eccebe757d88a0b84e4dbe314b968c8.